### ISSUE TITLE
Initial text for science page

### ISSUE THREAD

**Issue by snim2** [PyconUK/pyconuk.github.io#31]
I'm about to tidy up the text for the science page and push it, but unfortunately everything I have written so far was on a clone of the old repo on BitBucket. Now that everything has moved to Jekyll what I did before needs doing again. Can someone please tell me where assets go and how I use them? On the old site I was putting images in `.../science/static/img/`. I notice the Education page on the new Jekyll site doesn't have any images now, so I can't just copy what @ntoll has done :(

**Comment by drvinceknight** [comment]
Usually there's an asset directory in the root somewhere that you can get to with:

```
{{site.baseurl}}/assets/picture.png
```

I see that [/static/assets](https://github.com/PyconUK/pyconuk.github.io/tree/master/static/assets) exists so perhaps that's where things should go:

```
{{site.baseurl}}/static/assets/science/picture.png
```

(Sorry if I'm in the way and shouldn't be throwing in my two cents: please ignore me :+1: )

**Comment by snim2** [comment]
Thanks @drvinceknight - so does Jekyll infer that something with a `.png` extension will live in the `img` directory?

**Comment by drvinceknight** [comment]
Eeeeehhhh no not at all (saying that with 75% certainty :)), you can link to any file type just using the `{{site.baseurl}}/path` thing... Might be worth having a look at how other images have been put in for consistency... Running of to a class now, will check back in if I can be helpful :)

**Comment by inglesp** [comment]
Hi @snim2 -- do you want to have a quick hangout to look at this together?  Might be easier to do in realtime...

**Comment by snim2** [comment]
This sounds great, many thanks for the offer @inglesp . I have classes all day today until 5, I should be home some time just after 7 today, if that suits you?

**Comment by inglesp** [comment]
Yep -- let me know when's best.

**Comment by snim2** [comment]
Closed in PR #33
